Screens/Settings.tsx

```diff
@@ -0,0 +1,537 @@
+import React, { useState } from "react";
+import { StatusBar } from "expo-status-bar";
+import {
+    ScrollView,
+    StyleSheet,
+    Text,
+    TouchableOpacity,
+    View,
+    TextInput,
+} from "react-native";
+import { Formik } from "formik";
+import * as Yup from "yup";
+import SelectDropdown from "react-native-select-dropdown";
+import Checkbox from "expo-checkbox";
+import Seperator from "../Components/Seperator";
+import { Ionicons } from "@expo/vector-icons";
+import CustomButton from "../Components/CustomButton";
+import SwitchSelector from "react-native-switch-selector";
+import {
+    checkboxHitSlop,
+    fullDayTimes,
+    generateEndTimes,
+} from "../Helpers/helpers";
+
+type AvailableDay = { value: number; label: string; checked: boolean };
+
+type SettingsValues = {
+    businessName: string;
+    availableDays: AvailableDay[];
+    deposit: string;
+    regularHoursStart: string;
```

Comment:
I feel like this is not a great name. Maybe operatingHoursStart and operatingHoursEnd?